### ISSUE TITLE
cy.edgehandles is not a function

### ISSUE THREAD

**Issue by amherag** [cytoscape/cytoscape.js-edgehandles#66]
I'm getting this error, as in issue #61. I downloaded the latest release a few weeks ago, and it works, but the current latest isn't working. I also tried with other releases.

**Comment by maxkfranz** [comment]
I have the latest release working on both the demo page and in an app using npm/browserify.  I'll need more info to reproduce your issue.

**Comment by amherag** [comment]
Could you please tell me what information I can provide in order for you to reproduce the problem? I'm running Mac OS, I tried only the demo on Firefox, Safari and Chrome.

The version that is working for me is 2.11.1.

**Comment by maxkfranz** [comment]
Can you make a jsbin that reproduces your issue?

**Comment by amherag** [comment]
Thank you for the reply.

I copy-pasted the code on jsbin and it worked perfectly. For some reason I decided to look at the code at cytoscape-edgehandles.js and I saw something weird on my Emacs:

![screenshot 2015-12-09 23 56 33](https://cloud.githubusercontent.com/assets/404199/26424594/a6c07474-4086-11e7-87e5-8d4d138ab4bd.png)

That lightblue underscore. I deleted it and replaced it with a blank space, and now the demo works. I copy-pasted that character on http://www.babelstone.co.uk/Unicode/whatisit.html, and it tells me that underscore is `U+00A0 : NO-BREAK SPACE [NBSP]`, while a "normal" space is `U+0020 : SPACE [SP]`.
